ynput/ayon-core
# Max: reset scene and clear undo buffer when starting new scene

## Changelog Description
When user load assets and then start the new scene by `File -> New -> New All`, the scene isn't reset correctly. This PR is to fix this bug so that user can get nice and clean scene after starting new scene.

## Additional info
n/a

## Testing notes:
1. Launch Max
2. Load Asset
3. start the new scene by `File -> New -> New All`
4. Scene should be reset correctly
